assignment-2/assignment2_q2.rb

```diff
@@ -0,0 +1,71 @@
+=begin
+2. Tiny Instacart!
+Compose a simple terminal program using one or more of the common objects from Instacart (orders, drivers, batches, order_items, payments, etc)
+
+Design a desired workflow. ex. The entire flow of a driver from accepting a batch to delivery
+Using the tools and concepts from the completed room cleaner program, construct a self-running terminal program that successfully runs through your intended workflow.
+Test your program thoroughly to ensure that it runs as expected.
+Possible Extensions:
+
+Ruby offers alternatives to if/else including unless, while, and until.
+Set parameters for your program through user input
+Make sure to incorporate if/else and the appropriate operators.
+Look up how to use a ruby hash to create an array with multiple attributes per array element and run an each loop that checks the values of said attributes. ex. (`drivers = [{name: "Rose", role: "in-store-shopper"}, {name: "Donna", role: "shift-lead"}, {name: "Amelia Pond", role: "full-service"}, {name: "Martha Jones", role: "cashier"})
+=end
+
+def accept_batch
```

Comment:
this works but there is no way for a shopper to go off shift - they'll get batches forever! my original suggestion was to have `accept_batch` return a `true` or `false` to see if we should proceed to `deliver_batch` and `customer_receive`.